File: game-server/src/main/java/org/forfun/mmorpg/game/vip/model/VipRight.java
package org.forfun.mmorpg.game.vip.model;

import java.util.HashSet;
import java.util.Set;

public class VipRight {

    private int level;

    private int exp;

    private Set<Integer> rewardedIds = new HashSet<>();

    @Override
    public String toString() {
        return "VipRight [level=" + level + ", exp=" + exp + "]";
    }

    public int getLevel() {
        return level;
    }

    public void setLevel(int level) {
        this.level = level;
    }

    public int getExp() {
        return exp;
    }

    public void setExp(int exp) {
        this.exp = exp;
    }

    public Set<Integer> getRewardedIds() {
        return rewardedIds;
    }

    public void setRewardedIds(Set<Integer> rewardedIds) {
        this.rewardedIds = rewardedIds;
    }

}

File: game-server/src/main/kotlin/org/forfun/mmorpg/game/vip/model/VipRight.kt
package org.forfun.mmorpg.game.vip.model

import java.util.HashSet

class VipRight {

    var level: Int = 0
    var exp: Int = 0
    var rewardedIds: MutableSet<Int> = HashSet()

    override fun toString(): String {
        return "VipRight [level=$level, exp=$exp]"
    }
}